packages/components/src/components/Table/Table.js

```diff
@@ -135,7 +135,7 @@ const Table = props => {
                 <TableBody>
                   {!loading && dataRows.length === 0 && (
                     <TableRow>
-                      <TableCell colSpan={headers.length}>
+                      <TableCell colSpan={headers.length + 1}>
```

Comment:
should this be conditional on the presence of batch actions? i.e. column span would match headers length for a normal table, and add the +1 when batch actions are specified (which triggers the checkbox column)

Reply:
Good point - updated that now :)